src/main/resources/messages/messages_pt.properties

```diff
@@ -0,0 +1,8 @@
+welcome=Bem-vindo
+required=� necess�rio
```

Comment:
I think the binary bytes are wrong here? They should show up as accented letters and I'm just seeing (?).

Reply:
I could remove the accentuation, but it wouldn't be in accordance with the correct Portuguese language.
Could you attach a printout of how it appears to you, please?